demo/run.sh

```diff
@@ -60,30 +59,43 @@ function createHostFactoryToken() {
     client \
     /conjurinc/ansible/create_host_token.sh
 
-  echo '-----'
+  export HFTOKEN=$(<conjur-client-files/output/hftoken.txt)
 }
 
-function runAnsible() {
-  echo "Running Ansible playbook"
+function conjurizeTargetContainer() {
+  echo '-----'
+  echo "Conjurizing the target container with Ansible"
   echo '-----'
 
-  export HFTOKEN=$(<client-output/hftoken.txt)
-
-  ansible-playbook demo.yml
+  ansible-playbook conjurize-container.yml
+}
 
+function RetrieveSecretInTargetWithCli() {
+  echo '-----'
+  echo "Retrieving secret in target container with Conjur cli"
   echo '-----'
+
+  docker exec ansible-target bash -c /conjurinc/ansible/conjur_cli.sh
+}
+
+function RetrieveSecretInTargetWithSummon() {
+  docker exec ansible-target bash -c /conjurinc/ansible/summon.sh
 }
 
-function retrieveSecretInTarget() {
-  echo "Fetching secret in target container"
+function RetrieveSecretWithPlugin() {
+  echo '-----'
+  echo "Retrieving secret with conjur_variable lookup plugin"
   echo '-----'
 
-  docker exec ansible-target bash -c "echo Conjur identity is: && conjur authn whoami && echo && echo Conjur secret is: \
-    && conjur variable value password && echo"
+#  todo - remove once the machine can be conjurized with /etc privileges
+  api_key=$(docker-compose exec -T conjur rails r "print Credentials['cucumber:host:ansible-master'].api_key")
+  export CONJUR_AUTHN_API_KEY=${api_key}
 
-  echo '-----'
+  ansible-playbook conjur-plugin.yml
```

Comment:
If `ansible-playbook` isn't yet running in a container, we should get that going. `ansible-playbook` is not installed on our Jenkins executors.

docker-compose + https://github.com/ansible/ansible-container may be a good combo.

Reply:
Will do this when we start working on getting this to Jenkins